var/spack/repos/builtin/packages/py-rios/package.py

```diff
@@ -31,6 +32,8 @@ class PyRios(PythonPackage):
     version("1.4.11", sha256="b7ae5311f987b32f1afe1fabc16f25586de8d15c17a69405d1950aeada7b748e")
     version("1.4.10", sha256="6324acccc6018f9e06c40370bc366dc459890e8c09d26e0ebd245f6fd46dad71")
 
+    conflicts("^py-numpy@2:", when="@:2.0.1")
```

Comment:
Bonus if you could leave a comment above this with the PR fixing the issue in `rios`